src/constant/constant.ts

```diff
@@ -23,3 +23,9 @@ export enum LeaveTypes {
   Vacation = "vacation",
   Personal = "personal",
 }
+
+export enum LeaveRequestStatus {
+  Pending = "pending",
+  Accept = "accept",
```

Comment:
Correct terminology would be accepted and rejected.

Reply:
Fixed these terminology